doc/source/cluster/vms/references/ray-cluster-configuration.rst

```diff
@@ -1408,21 +1408,31 @@ The minimum number of workers to maintain for this node type regardless of utili
 * **Minimum:** ``0``
 * **Maximum:** Unbounded
 
-.. _cluster-configuration-node-max-workers:
-
-``available_node_types.<node_type_name>.node_type.max_workers``
+``available_node_types.<node_type_name>.node_type.min_workers``
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The maximum number of workers to have in the cluster for this node type regardless of utilization. This takes precedence over :ref:`minimum workers <cluster-configuration-node-min-workers>`. By default, the number of workers of a node type is unbounded, constrained only by the cluster-wide :ref:`max_workers <cluster-configuration-max-workers>`. (Prior to Ray 1.3.0, the default value for this field was 0.)
+Deprecated since Ray 2.9 in favor of :ref:`min_worker_nodes <cluster-configuration-node-min-worker-nodes>`, which has identical behavior but has a new name to avoid confusion with worker processes.
+
+.. _cluster-configuration-node-max-worker-nodes:
+
+``available_node_types.<node_type_name>.node_type.max_worker_nodes``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The maximum number of worker nodes to have in the cluster for this node type regardless of utilization. This takes precedence over :ref:`minimum workers <cluster-configuration-node-min-worker-nodes>`. By default, the number of worker nodes of a node type is unbounded, constrained only by the cluster-wide :ref:`max_worker_nodes <cluster-configuration-max-worker-nodes>`. (Prior to Ray 1.3.0, the default value for this field was 0.)
 
 Note, for the nodes of type ``head_node_type`` the default number of max workers is 0.
```

Comment:
Is this supposed to be max worker nodes?

Reply:
Yup, fixed